lib/clickhouse_ecto/query_string.ex

```diff
@@ -258,10 +263,8 @@ defmodule ClickhouseEcto.QueryString do
     Connection.all(query)
   end
 
-  def expr({:fragment, _, [kw]}, sources, query) when is_list(kw) or tuple_size(kw) == 3 do
-    Enum.reduce(kw, query, fn {key, {op, val}}, query ->
-      expr({op, nil, [key, val]}, sources, query)
-    end)
+  def expr({:fragment, _, [kw]}, _sources, query) when is_list(kw) or tuple_size(kw) == 3 do
+    Helpers.error!(query, "ClickHouse adapter does not support keyword or interpolated fragments")
```

Comment:
@HammamSamara One of the tests were failing on a `FunctionClauseError` related to this function head. I've reverted to **clickhouse-elixir/clickhouse_ecto** to check their implementation to aid fix the issue https://github.com/clickhouse-elixir/clickhouse_ecto/blob/master/lib/clickhouse_ecto/query_string.ex#L248-L250

[Note]: [Keyword Fragments](https://github.com/clickhouse-elixir/clickhouse_ecto/blob/master/lib/clickhouse_ecto/query_string.ex#L248-L250) are meant to support databases that don't have string-based queries, like MongoDB, and is not supported (i.e. is irrelevant) in case of Clickhouse adapter, so triggering a QueryError here is the right thing to do indeed!